packages/components/plh/plh-kids-kw/components/activity-check-in/activity-check-in.component.html

```diff
@@ -13,16 +14,14 @@
             }"
           ></div>
         </div>
-        <div class="countdown-text">{{ countdownText() }}</div>
       </div>
     }
-
     <div class="image">
       <img [src]="params.lockedImageAsset | plhAsset" />
     </div>
     <div class="details">
       <div class="title">
-        <h4>{{ params.title }}</h4>
+        <p>{{ params.title }}</p>
```

Comment:
nit(non-blocking): There seems to be an unecessary level of nesting here which can sometimes lead to inflexibility with styles (the div is redundant, it could be the `p` element itself, or the title could just be text in the div. But not a major issue